src/common/interflop.h

```diff
@@ -21,6 +31,37 @@ enum FCMP_PREDICATE {
   FCMP_TRUE,
 };
 
+typedef struct interflop_function_info
+{
+  // Indicate if the function is from library
+  char id[500];
+  // Indicate if the function is from library
+  char isLibraryFunction;
+  // Indicate if the function is intrinsic
+  char isIntrinsicFunction;
+  // Indicate if the function use float
+  char haveFloat;
+  // Indicate if the function use float
```

Comment:
use double ?

I'm not sure to like haveFloat / haveDouble... I image they are used to only call backends for functions which have float/double arguments or return ?

Reply:
havefloat and haveDouble are now useFloat and useDouble